src/model.jl

```diff
@@ -134,11 +140,11 @@ end
 
 
 function _make_sets(case::Case)::Sets
-    _make_sets(case, nothing)
+    _make_sets(case, nothing, nothing)
 end
 
 
-function _make_sets(case::Case, storage::Union{Storage,Nothing})::Sets
+function _make_sets(case::Case, storage::Union{Storage,Nothing}, demand_flexibility::Union{DemandFlexibility, Nothing})::Sets
```

Comment:
I wonder if we could combine these:
```julia
_make_sets(
    case::Case,
    storage::Union{Storage,Nothing}=nothing,
    demand_flexibility::Union{DemandFlexibility, Nothing}=nothing,
)::Sets
```

Reply:
In a similar vein, it might be nice to call `_make_sets` in `REISE.jl` (within the `run_scenario` function) so that the sets only need to be created once. Right now, we create the sets in `loop.jl`, `model.jl`, and `query.jl`. I don't believe the information in the sets changes as the simulation progresses.